tests/artifacts/README.MD

```diff
@@ -0,0 +1,35 @@
+# Test artifacts
+
+This directory contains test artifacts for the `readlif` package. These artifacts consist of example LIF files from various sources, sometimes paired with expected outputs in the form of individual TIFF files.
+
+## About the LIF files
+
+### `xyzt-example/xyzt-example.lif`
+
+Contains a single image. 1024 x 1024. 8-bit. Shape: C2, T4, Z3.
```

Comment:
Yeah, the `dims` attribute contains all of the dimensions other than the channel dimension, which appears in the separate `channels` attribute. I *think* this is because the channels are stored separately in the LIF metadata, but I'm not sure.

Reply:
Also, `dims` only contains `XYZTM`, so it doesn't include the lambda-scan-related dimensions (excitation and emission wavelengths).